app/controllers/api/v1x0/templates_controller.rb

```diff
@@ -2,6 +2,9 @@ module Api
   module V1x0
     class TemplatesController < ApplicationController
       include Mixins::IndexMixin
+      include Mixins::RBACMixin
+
+      before_action :read_access_check, :only => %i[show index]
```

Comment:
you already include `index` in `before_action`, do you still need #152 for templates_controller?

Reply:
Will remove `index`